packages/lib/server/getLuckyUser.ts

```diff
@@ -221,7 +221,7 @@ export async function getLuckyUser<
     return availableUsers[0];
   }
 
-  const bookingsOfAvailableUsers = await BookingRepository.getAllBookingsForRoundRobin({
+  const bookingsOfAvailableUsers = await BookingRepository.getBookingsForRoundRobin({
```

Comment:
`bookingsOfAvailableUsers` is passed to `getUsersBasedOnWeights` and to `leastRecentlyBookedUser`, but for `getUsersBasedOnWeights` we only need bookings for the last 28 days and for `leastRecentlyBookedUser` we need all bookings

Reply:
updated accordingly